src/Microsoft.Data.SqlClient/netcore/src/Resources/SR.Designer.cs

```diff
@@ -10,16 +10,16 @@
 
 namespace System {
     using System;
-
-
+    
+    
```

Comment:
Remove extra line.

Reply:
It's an auto-generated file, so I guess these changes from Visual Studio itself. Removing/fixing anything here will always modify this file eventually..